### ISSUE TITLE
about def get_masks(dataset_name, num_sem_pos=0, num_top_pos=0)： function

### ISSUE THREAD

**Issue by EtAior5** [Zehong-Wang/HGCML#3]
```
sem_pos = np.load(sem_pos_path)[:, :num_sem_pos]
top_pos = np.load(top_pos_path)[:, :num_top_pos]
```

These two values num_sem_pos and num_top_pos use the default value of 0, Then they get the same identity matrix through function 
`def idx2mask(idx):`

So, in the main function, `mask = torch.logical_or(sem_mask, top_mask).to(device)`  seems to have no effect.
I'm not sure if there are any issues with my understanding. I hope to receive your reply.
